ElasticaAdmin/BlockParameter/ElasticSearchStrategy.php

```diff
@@ -10,6 +10,8 @@
  */
 class ElasticSearchStrategy implements BlockParameterInterface
 {
+    const NAME = 'elastica_search';
```

Comment:
I actually did not alter the getName to remain coherent as it is done in others classes NicolasT is agree with that.